ee/clickhouse/views/test/test_clickhouse_experiments.py

```diff
@@ -782,6 +837,116 @@ def test_experiment_flow_with_event_results(self):
         self.assertEqual(response_data["significance_code"], ExperimentSignificanceCode.NOT_ENOUGH_EXPOSURE)
         self.assertAlmostEqual(response_data["expected_loss"], 1, places=2)
 
+    @pytest.mark.skip(
+        reason="This test needs proper timezone support, coming in https://github.com/PostHog/posthog/pull/11935"
+    )
```

Comment:
This can be addressed now?

Reply:
Oh, lovely, missed that it had been merged! 👀

... Is that why you were waiting to review 😂